src/cairo_interaction.rs

```diff
@@ -10,11 +10,11 @@ use glib::object::IsA;
 use glib::translate::*;
 use {Rectangle, Window, RGBA};
 
-pub trait SurfaceExt {
+pub trait GdkSurfaceExt {
```

Comment:
"Gdk"SurfaceExt?

Reply:
It comes from GDK but do we really want to name it like this?

Reply:
IMHO yes, as  "it comes from GDK"

Reply:
Yes?

Reply:
Fine then

Reply:
Just like we have GtkWindowExt for example

Reply:
This is all too complicated for me.